package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -247,8 +249,8 @@ class DistanceMatrix(AnalysisBase):
     .. versionchanged:: 2.2.0
          :class:`DistanceMatrix` now also accepts `AtomGroup`.
     """
-    def __init__(self, universe, select='all', metric=rmsd, cutoff=1E0-5,
-                 weights=None, **kwargs):
+    def __init__(self, universe: Union[Universe, AtomGroup], select: str = 'all', metric=rmsd, cutoff: float = 1E0-5,
```

Comment:
```suggestion
    def __init__(self, universe: Union[Universe, AtomGroup], select: str = 'all', metric: Callable[..., float] = rmsd, cutoff: float = 1E0-5,
```

I tried being more precise with the annotation of `method`, but I did not figure out something that would accept anything that can be called there and still accept more aguments: https://github.com/MDAnalysis/mdanalysis/blob/5e8d5f6f957418e3ca74812b6c48870c78e0bbd5/package/MDAnalysis/analysis/diffusionmap.py#L282